internal/bot/bot.go

```diff
@@ -57,58 +45,55 @@ func main() {
 
 	bot, err := tg.NewBotAPI(cfg.BotToken)
 	if err != nil {
-		log.Error("Failed connect to Telegram.")
-		log.Error(err)
+		log.Errorf("Failed connect to Telegram. %v", err)
 		os.Exit(3)
 	}
+	log.Infof("Authorized on telegram account @%s", bot.Self.UserName)
+
 	bot.Debug = cfg.TelegramDebug
 
-	app := App{
+	return &t.Bot{
 		Config: cfg,
 		DB:     db,
-		Bot:    bot,
+		API:    bot,
+		Log:    log,
 	}
+}
 
-	log.Infof("Authorized on telegram account @%s", bot.Self.UserName)
-
+// Start starts polling for messages for bot
+func Start(bot *t.Bot) {
 	u := tg.NewUpdate(0)
 	u.Timeout = 60
 
-	updates, err := bot.GetUpdatesChan(u)
+	updates, err := bot.API.GetUpdatesChan(u)
+
+	if err != nil {
+		bot.Log.Errorf("Failed get updates from Telegram. %v", err)
+		os.Exit(4)
```

Comment:
Why status code is `4`?
https://tldp.org/LDP/abs/html/exitcodes.html

Reply:
I just used incremental value for exit code values. No particular reason.
Fixed in 9fd8bdf67a